includes/class-wc-siftscience-html.php

```diff
@@ -27,6 +27,119 @@ class WC_SiftScience_Html {
 		public const WC_CHECKBOX_ELEMENT   = 'checkbox';
 		public const WC_SECTIONEND_ELEMENT = 'sectionend';
 		public const WC_CUSTOM_ELEMENT     = 'custom';
+		
+		public function __call( $name, $args ) {
+			if ( 'create_element' === $name ) {
+
+				switch ( count( $args ) ) {
+					case 2:
+						return $this->create_element( $args[0], $args[1], '', '', array() );
+					case 3:
+						return $this->create_element( $args[0], $args[1], $args[2], '', array() );
+					case 4:
+						return $this->create_element( $args[0], $args[1], $args[2], $args[3], array() );
+					case 5:
+						return $this->create_element( $args[0], $args[1], $args[2], $args[3], $args[4] );
+					default:
+						$this->logger->log_error( 'there is no delaretion method for create_element with ' . count( $args ) . ' arguemnts' );
+						break;
+				}
+			}
+		}
+
+		/**
+		 * This function sets HTML element attributes according to woocommearce provided library.
+		 * desc_tip Mixed [bool:false] (default)
+		 *     field type of checkbox; the desc text is going next to the control
+		 *     field type of select, number or text; the desc text is going underneath control
+		 * desc_tip Mixed [bool:true]
+		 *     [X] field type of checkbox; the desc text is going underneath control
+		 *     field type of select, number or text; a question mark pop-up appears before control with desc text
+		 * desc_tip Mixed [string]
+		 *     field type of checkbox; the desc_tip text is going underneath control
+		 *     field type of select, number or text; a question mark pop-up appears before control with desc_tip text
+		 * NOTE: if desc_tip isn't a string and it's a checkbox [X] disc_tip is sanitized to false, add it in element_options
+		 *
+		 * @param string $type            Element type name.
+		 * @param string $id              HtmlElement ID.
+		 * @param string $label           Element label.
+		 * @param string $desc            Description text.
+		 * @param array  $element_options Element special options.
+		 *
+		 * @return array $element         An array of attributes.
+		 * @since 1.1.0
+		 */
+		private function create_element( $type, $id, $label, $desc, $element_options ) {
```

Comment:
oh, it's private. Interesting. Now I'm not sure.

Reply:
i'm sure